src/main/java/seedu/address/model/remark/Remark.java

```diff
@@ -13,32 +13,26 @@
  * Represents a Point of contact in the client.
```

Comment:
Great job refactoring! I think you might have missed out on this class description. Maybe can remove that it represents a point of contact. All the best!

Reply:
Thank you for spotting this mistake! I'll make the changes.